personalize.module

```diff
@@ -1100,6 +1100,17 @@ function personalize_agent_set_status($agent_name, $status) {
     $end_time_var = _personalize_agent_get_stoptime_variable($agent_name);
     $now = time();
     variable_set($end_time_var, $now);
+    // Check if we need to delete all the actions for completed campaigns.
```

Comment:
Yes, definitely - good catch, will make that change now. I need to recreate the PR because it was originally done against the RC branch, not sure if these comments will get blown away.